### PR TITLE
Add Storage docs notebook

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: files/var/notebooks/Storage.ipynb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (mariyadavydova) [inline files/var/notebooks/Storage.ipynb:1]: Let's call this file "Help".
I would like to add more useful info later, like "How long is the notebook session alive" etc.

File: targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile [hunk 2]
```diff
@@ -236,6 +236,12 @@ PermitEmptyPasswords yes\n" > /etc/ssh/sshd_config
 # ssh port
 EXPOSE 22
 
+# ==================================================================
+# Documentation notebook
+# ------------------------------------------------------------------
+RUN mkdir -p /var/notebooks
+COPY files/var/notebooks/Storage.ipynb /var/notebooks
```
Comment (mariyadavydova) [inline targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:243]: I'm slightly lost here. I thought that we agreed that this file is copied to /var/storage (if there is no one there yet), but I don't see where does it happen.

File: files/var/notebooks/Storage.ipynb (hunk 1 above)
Comment (anayden) [reply files/var/notebooks/Storage.ipynb:1]: Ok, will update now

Comment (yartem) [reply files/var/notebooks/Storage.ipynb:1]: or `README.ipynb`

File: targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile (hunk 2 above)
Comment (anayden) [reply targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:243]: It's copied to the image and then, when user runs Jupyter from web UI `platform-pipelines` copy the file to storage.

Comment (mariyadavydova) [reply targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:243]: Ah, OK, thanks.

File: files/var/notebooks/Storage.ipynb (hunk 1 above)
Comment (mariyadavydova) [reply files/var/notebooks/Storage.ipynb:1]: Agree, README is more expected here.

**Pull request merged**